app/controllers/api/v1/agencies_controller.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+# API for agencies.  Note that this class is 'read only' for client apps.
+#   Data required for Agencies will be added to the db via rake tasks or other external means.
```

Comment:
Make sure you remove this file since we decided in #170 not to have controller actions